src/main/java/dev/jbang/net/JdkProvider.java

```diff
@@ -121,6 +121,8 @@ default Jdk createJdk(@Nonnull String id, @Nullable Path home, @Nonnull String v
 
 	default String name() {
 		String nm = getClass().getSimpleName();
+		// TODO: this 11 is here assuming it ends in "JdkProvider" - dont make that
```

Comment:
Why not? It's a reasonable default implementation, isn't it? If your implementation doesn't follow that scheme you can simply override this method and return the value that you desire.

Reply:
I had a name called LinxDistroProvider and it showed up as 'linuxdist' which confused me - will update one that is a bit more explicit.